deps/eventlog/src/lib.rs

```diff
@@ -117,12 +117,12 @@ impl CcEventLog {
         let digest_map = collect_digests_by_index(&self.log);
 
         for item in data.iter() {
-            let calculated_ccel_rtmr = replay(&digest_map, item.index, item.algorithm)?;
-            if calculated_ccel_rtmr != item.reference {
+            let calculated_ccel_ccmr = replay(&digest_map, item.index, item.algorithm)?;
+            if calculated_ccel_ccmr != item.reference {
                 bail!(
-                    "CCEL eventlog does not pass RTMR [{}] check. CCEL value: {}, Quote value: {}",
+                    "CCEL eventlog does not pass CCMR [{}] check. CCEL value: {}, Quote value: {} \nThe relation ship between CCMR and RTMR can be found in https://uefi.org/specs/UEFI/2.10/38_Confidential_Computing.html#vendor-specific-information",
```

Comment:
But the error is not related to vendor mapping here, is it? You could extend an MR without writing a log entry for it and it'll also fail here.

Reply:
Maybe I misunderstood that this refers to that PCR to MR "relationship" / mapping. Could we just say MR or measurement register? For example the kernel uses TDX measurement register (MR) wording.

Reply:
I added a comment upon the `index` field of `ReferenceMeasurement` struct.